manage-teams-identity-mobile-and-desktop/README.md

```diff
@@ -0,0 +1,55 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+- azure-communication-services
+---
+
+# Create and manage Communication access tokens for Teams users in mobile and desktop applications
+
+This code sample walks you through the process of acquiring a Communication Token Credential by exchanging an Azure AD token of a user with a Teams license for a valid Communication access token.
+
+This sample application utilizes the [MSAL.js v2.0](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node) (`msal-node`) package for authentication against the Azure AD and acquisition of a token with delegated permissions and [Express.js](https://expressjs.com/). The token exchange itself is facilitated by the `@azure/communication-identity` package.
+
+The initialization of a Communication credential object that can be used for Calling is achieved by the `@azure/communication-common` package.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- Node.js [Active LTS version](https://nodejs.org/en/about/releases/).
+- An active Communication Services resource and connection string. [Create a Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource/).
+- Azure Active Directory tenant with users that have a Teams license.
+
+## Before running sample code
+
+1. Complete the [Administrator actions](https://docs.microsoft.com/azure/communication-services/quickstarts/manage-teams-identity?pivots=programming-language-javascript#administrator-actions) from the [Manage access tokens for Teams users quickstart](https://docs.microsoft.com/azure/communication-services/quickstarts/manage-teams-identity).
+   - Take a not of Fabrikam's Azure AD Tenant ID and Contoso's Azure AD App Client ID. You'll need the values in the following steps.
+1. On the Authentication pane of your Azure AD App, add a new platform of the mobile and desktop application type with the Redirect URI of `http://localhost:3000/refirect`.
```

Comment:
```suggestion
1. On the Authentication pane of your Azure AD App, add a new platform of the mobile and desktop application type with the Redirect URI of `http://localhost:3000/redirect`.
```

Reply:
you wanna do the `:3000` port here? the tutorial doesn't mention it. we could probably go without it so that the code is 100% the same

Reply:
nope I played around on that port, and wanted to leave it as a default, but you are right I will remove it